x-pack/heartbeat/monitors/browser/config.go

```diff
@@ -32,6 +32,7 @@ type Config struct {
 	Throttling        interface{}                   `config:"throttling"`
 	Screenshots       string                        `config:"screenshots"`
 	SyntheticsArgs    []string                      `config:"synthetics_args"`
+	PlaywrightOpts    map[string]interface{}        `config:"playwright_options"`
```

Comment:
nit: mapstr.M

Reply:
TBH, I kinda prefer ignoring our weird types unless we actual have a need for them. It doesn't make a difference here since we never traverse it, just serialize it, so I like to keep it plain.